runtime/src/lib.rs

```diff
@@ -2012,6 +2012,25 @@ impl order_book::Config for Runtime {
     type WeightInfo = order_book::weights::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+
```

Comment:
redundant empty line

Reply:
Resolved in https://github.com/sora-xor/sora2-network/pull/512/commits/50e06c86051720b8bde516dbbb2abf2a474926eb